tests/third_party/osmosis_testnet/net_config.py

```diff
@@ -18,42 +18,44 @@
 # ------------------------------------------------------------------------------
 
 """Osmosis network config."""
-from time import sleep
-
-import requests
-
+from cosmpy.aerial.client import LedgerClient
 from cosmpy.aerial.config import NetworkConfig
+from cosmpy.aerial.wallet import LocalWallet
+
 
+# Network config
+PREFIX = "osmo"
+COIN = "uosmo"
+GAS_LIMIT = 120000
```

Comment:
This is here to override inherited params - I tried to init with super.init but pytests weren't working with it. - But I agree this is not the best way of doing it.